server/models/food.js

```diff
@@ -1,43 +1,44 @@
 import mongoose from 'mongoose'
 
-import {modelTypes} from '../../common/constants'
+import { modelTypes } from '../../common/constants'
 
-const {Schema} = mongoose
+const { Schema } = mongoose
 
 const FoodItemSchema = new Schema({
-  name: {
-    type: String,
-    trim: true
-  },
-  quantity: {
-    type: Number,
-    default: 0
-  },
-  startDate: {
-    type: Date
-  },
-  frequency: {
-    type: Number,
-    default: 0
-  },
-  deleted: {
-    type: Boolean,
-    default: false
-  }
+    name: {
```

Comment:
Looks good, but now the indentation's wrong. You might have to set your editor to use 2 spaces for tabs.